src/Sword.java

```diff
@@ -0,0 +1,37 @@
+
+/**
+ * Created by connorparrott123 on 12/12/2016.
+ */
+public class Sword extends Weapon {
+    boolean isBroken = false;
+
+    public Sword(String name, String description) {
+        super(name, description, 15);
+    }
+
+    @Override
+    public boolean useOn(Character other) {
```

Comment:
this method doesn't prevent you from using your weapon while it's broken. add an if statement to do that